### ISSUE TITLE
Not setting password will not default to jupyter

### ISSUE THREAD

**Issue by a-nldisr** [dcos-labs/dcos-jupyterlab-service#19]
Left the jupyter_password empty, started up the jupyterlab and could not log in with the default according to the help. Thus the password does not default to jupyter.

**Comment by vishnu2kmohan** [comment]
@a-nldisr Did you deploy the service into a folder - e.g., `/dev/jupyterlab-notebook`?

If so, the password would change to [`jupyter-dev`](https://github.com/dcos-labs/dcos-jupyterlab-service/blob/master/jupyter_notebook_config.py#L17-L18): 

```
    # Set Jupyter Notebook Server password to 'jupyter-<Marathon-App-Prefix>'
    # e.g., Marathon App ID '/foo/bar/app' maps to password: 'jupyter-foo-bar'
```

Thanks for the feedback, I'm already working on improving the description for the [config option](https://github.com/dcos-labs/dcos-jupyterlab-service/blob/master/universe/jupyterlab/1/config.json#L427) for the next release.

**Comment by a-nldisr** [comment]
Indeed i put it in the folder called jupyterlab.

**Comment by a-nldisr** [comment]
Shall i leave this open till this is merged? Maybe best way to communicate is to add this to the README for now instead?

**Comment by vishnu2kmohan** [comment]
@a-nldisr We've released a new version [`1.3.0-0.35.4`](https://github.com/dcos-labs/dcos-jupyterlab-service/releases/tag/1.3.0-0.35.4) of the service and it's now renamed to `beta-mesosphere-jupyter-service` in the DC/OS Catalog 

Please give this new version a spin and let me know if it resolves your issues.
